### DISCUSSION TITLE
Nuxt3 Support

### DISCUSSION THREAD

**Discussion by narr07** [ivodolenc/nuxt-animejs#10] [Q&A] [answered]
Nuxt3 not in beta

**Comment by ivodolenc** [comment]
To work properly in Nuxt 3, the module needs to be completely rewritten in typescript, and this takes a lot of time, mostly for docs, types, and examples.

I don't have much time to do it for free at the moment. If enough people can be gathered who would like to sponsor this project, I am willing to invest the time to update it to the latest version with proper support.

Please understand that I also need to somehow compensate for the time spent on this project. Anyway, thank you all for your interest.

This is an open source project so anyone can fork it or create a new version on top of it.

**Reply by CraftingGamerTom** [reply]
I've recently hit the wall of the Nuxt3 module drought. AnimeJS is on my list of used modules. Perhaps I can help? I'll take a look at [nuxt-animejs](https://github.com/ivodolenc/nuxt-animejs) later this week and create a fork. Am I correct in assuming nothing has been done yet to migrate? or should I be starting from somewhere other than the main branch?

**Comment by ivodolenc** [comment, marked as answer]
Hi guys, I finally updated the module so feel free to try the new version.

The module is optimized and supports nuxt 3 with typescript. Have fun!

**Reply by narr07** [reply]
Thanks bro, amazing 🤩
